js/packages/components/main/CreateGroupFinalize.tsx

```diff
@@ -226,29 +225,31 @@ export const CreateGroupFinalize: React.FC = () => {
 	}, [done, error, reset, reply, dispatch])
 	return (
 		<Layout style={[flex.tiny]}>
-			<View style={{ backgroundColor: colors['background-header'] }}>
-				<View onLayout={e => setLayout(e.nativeEvent.layout.height)}>
-					<MemberList />
-					<Header
-						title={t('main.home.create-group.add-members')}
-						onPress={() => goBack()}
-						style={[padding.bottom.small]}
-						first
-					/>
-				</View>
-				<View style={[{ top: -5, paddingBottom: insets.bottom }]}>
-					<Header title={t('main.home.create-group.group-info')}>
-						<GroupInfo onGroupNameChange={setGroupName} layout={layout} />
-					</Header>
-				</View>
-			</View>
-			<FooterCreateGroup
-				title={t('main.home.create-group.create-group')}
-				action={() => {
-					createGroup()
-					ctx.playSound('groupCreated')
-				}}
-			/>
+			<KeyboardAvoidingView behavior='position'>
```

Comment:
be careful with keyboardavoiding on android, the os already has a system like that and the rn one seems to have bugs